packages/server-wallet/src/wallet/index.ts

```diff
@@ -719,14 +746,21 @@ export class Wallet extends EventEmitter<WalletEvent>
 
   // ChainEventSubscriberInterface implementation
   holdingUpdated(arg: HoldingUpdatedArg): void {
-    const channelUpdate: WalletEventName = 'channelUpdate';
+    const channelUpdate: ChannelUpdateEventName = 'channelUpdate';
```

Comment:
nit: I think we have used "ChannelUpdat**ed**" in the past and this is what you'd find in the `client-api-schema`.

Reply:
👍 a4b8dd1c5